content/Exception Tracking - Tiziana Romani.md

```diff
@@ -0,0 +1,43 @@
+# Exception Tracking
+###### Production exceptions are crucial. We don't want errors encountered by clients or crashes in our application to go unnoticed, so keeping track of them it's a must! 
+
+[![Tulips](https://get.wallhere.com/photo/1920x1200-px-bokeh-fields-flowers-sea-tulips-white-1744269.jpg)](https://wallhere.com/es/wallpaper/1744269)
+
+An increase in software applications needs is appreciable nowadays. People want software to make life easier, thus the user experience must be rated as FFF: friendly, fast, and fine. Easy right? Not.
+Every developer wishes their software to work like a charm, but since developers are human (at least for now), it is known that humans make mistakes. 
```

Comment:
```suggestion
Every developer wishes their software to work like a charm, but since developers are human (at least for now), it is known for them to make mistakes. 
```